### PR TITLE
Change EMC update source branch to m̶a̶s̶t̶e̶r̶ main

### PR DESCRIPTION
**Pull request by matentzn**
Depends on 

- [ ] https://github.com/monarch-initiative/mondo-ingest/pull/628

Do not merge before that is merged to `master` (NOT just `develop`!)

### CODE REVIEW

File: src/ontology/mondo.Makefile [hunk 1]
```diff
@@ -274,8 +274,7 @@ $(TEMPLATES_DIR)/ROBOT_addMedGen_fromIngest.tsv:
 # 1. Rare disease pipeline
 # 2. Externally managed content pipeline
 
-# CHANGE THIS TO THE MAIN BRANCH BEFOR MERGING!!!
-MONDO_INGEST_EXTERNAL_LOCATION=https://raw.githubusercontent.com/monarch-initiative/mondo-ingest/externalclingenmedgenefo/src/ontology/external
+MONDO_INGEST_EXTERNAL_LOCATION=https://raw.githubusercontent.com/monarch-initiative/mondo-ingest/master/src/ontology/external
```
Comment (twhetzel) [inline src/ontology/mondo.Makefile:277]:
```suggestion
MONDO_INGEST_EXTERNAL_LOCATION=https://raw.githubusercontent.com/monarch-initiative/mondo-ingest/main/src/ontology/external
```

**Review by twhetzel — APPROVED** [review]
I updated the new `MONDO_INGEST_EXTERNAL_LOCATION` from `master` to `main`.

**Comment by twhetzel** [comment]
@matentzn can this be changed to `main` now?

**Comment by matentzn** [comment]
Relevant changes are still on develop?

**Pull request merged**